allexperiments.go

```diff
@@ -139,6 +140,20 @@ var experimentsByName = map[string]func(*Session) *ExperimentBuilder{
 		}
 	},
 
+	"http_host_header": func(session *Session) *ExperimentBuilder {
+		return &ExperimentBuilder{
+			build: func(config interface{}) *Experiment {
+				return NewExperiment(session, httphost.NewExperimentMeasurer(
+					*config.(*httphost.Config),
+				))
+			},
+			config: &httphost.Config{
+				TestHelperURL: "http://example.com",
+			},
```

Comment:
```suggestion
			config: &httphost.Config{},
```

As stated below, I'd rather configure the default inside the experiment directory itself.